.golangci.yml

```diff
@@ -1,59 +1,5 @@
-# © Broadcom. All Rights Reserved.
-# The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
-# SPDX-License-Identifier: MPL-2.0
-
-# For more information about the golangci-lint configuration file, refer to:
-# https://golangci-lint.run/usage/configuration/
-
-issues:
-  exclude-rules:
-    - path: _test\.go
-      linters:
-        - dogsled
-        - funlen
-        - gocognit
-    - path: internal/authctx
-      linters:
-        - gosec
-        - nestif
-    - path: flatten_test.go
-      linters:
-        - dupl
-
-linters:
-  disable-all: true
-  enable:
-    - asciicheck
-    - bodyclose
-    - copyloopvar
-    - dogsled
-    - dupl
-    - errcheck
-    - funlen
-    - goconst
-    - gocognit
-    - gocritic
-    - gocyclo
-    - godot
-    - gofmt
-    - goimports
-    - goprintffuncname
-    - gosec
-    - gosimple
-    - govet
-    - ineffassign
-    - misspell
-    - nakedret
-    - nestif
-    - prealloc
-    - staticcheck
-    - stylecheck
-    - typecheck
```

Comment:
stylecheck and typecheck have also been removed on purpose?

Reply:
Reference: 
- https://github.com/golangci/golangci-lint/pull/5487
- https://github.com/golangci/golangci-lint/pull/5543